splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -28,6 +28,9 @@
 logger = logging.getLogger("ucc_gen")
 
 
+LIBS_REQUIRED_FOR_UI = {"splunktaucclib": "6.4"}
```

Comment:
Is there a reason to have it `6.4` and not `6.4.0`?

Reply:
there is none, from the Version() method there is no difference but I changed it for accuracy